### PR TITLE
dune build @runtest -> dune runtest

### PR DESCRIPTION
**Pull request by emillon**
This replaces `dune build @runtest` by `dune runtest` in the tutorial. This command is equivalent and the recommended short form to run tests.

### CODE REVIEW

**Review by cuihtlauac — APPROVED** [review]
Thanks!

File: data/workflows/07_run_tests.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ with a module `dummy_test.ml`:
 let () = exit 1
 ```
 
-Running `dune build @runtest` will fail with the following output:
+Running `dune runtest` will fail with the following output:
```
Comment (tmattio) [inline data/workflows/07_run_tests.md]:
```suggestion
Running `dune test` will fail with the following output:
```

File: data/workflows/07_run_tests.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ let () =
   Alcotest.run "Dummy" [ "Greeting", suite ]
 ```
 
-If we run `dune build @runtest` again, the test should be successful and output the following:
+If we run `dune runtest` again, the test should be successful and output the following:
```
Comment (tmattio) [inline data/workflows/07_run_tests.md]:
```suggestion
If we run `dune test` again, the test should be successful and output the following:
```

**Review by tmattio — COMMENTED** [review]
What's the recommendation between `dune test` and `dune runtest`? I tend to favour the former.

**Comment by emillon** [comment]
I use `runtest` because it maps to the internal name. The Dune docs use `runtest`. As I understand there's no plan to remove either name and they're equivalent.

File: data/workflows/07_run_tests.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: "Running tests"
 
 > **TL;DR**
 > 
-> Add a `test` stanza in your dune file and run the tests with `dune build @runtest`.
+> Add a `test` stanza in your dune file and run the tests with `dune runtest`.
```
Comment (christinerose) [inline data/workflows/07_run_tests.md]:
```suggestion
> Add a `test` stanza in your `dune` file and run the tests with `dune runtest`.
```

**Review by christinerose — APPROVED** [review]
Just one single formatting fix. 🙂

**Review by christinerose — APPROVED** [review]

**Review by sabine — APPROVED** [review]

File: data/tutorials/platform/bp_03_run_executables_and_tests.md [hunk 4]
```diff
@@ -42,10 +42,9 @@ You can run it with `dune exec bin/main.exe` or `dune exec my-app`.
 
 ## Running Tests
 
-
 > **TL;DR**
 > 
-> Add a `test` stanza in your `dune` file and run the tests with `dune build @runtest`.
+> Add a `test` stanza in your `dune` file and run the tests with `dune runtest`.
```
Comment (sabine) [inline data/tutorials/platform/bp_03_run_executables_and_tests.md]:
```suggestion
> Add a `test` stanza in your `dune` file and run the tests with `dune test`.
```
